### ISSUE TITLE
Could not load file or assembly 'FontAwesome.WPF

### ISSUE THREAD

**Issue by RedTailedHawk** [DaxStudio/DaxStudio#304]
Just downloaded DaxStudio source, and when I run from Visual Studio I get the following:

```
System.Windows.Markup.XamlParseException
  HResult=0x80131501
  Message=Could not load file or assembly 'FontAwesome.WPF, PublicKeyToken=0758b07a11a4f466' or one of its dependencies. The system cannot find the file specified.
  Source=PresentationFramework
  StackTrace:
   at System.Windows.Markup.XamlReader.RewrapException(Exception e, IXamlLineInfo lineInfo, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.Load(XamlReader xamlReader, IXamlObjectWriterFactory writerFactory, Boolean skipJournaledProperties, Object rootObject, XamlObjectWriterSettings settings, Uri baseUri)
   at System.Windows.Markup.WpfXamlLoader.LoadBaml(XamlReader xamlReader, Boolean skipJournaledProperties, Object rootObject, XamlAccessLevel accessLevel, Uri baseUri)
   at System.Windows.Markup.XamlReader.LoadBaml(Stream stream, ParserContext parserContext, Object parent, Boolean closeStream)
   at DaxStudio.UI.Views.RibbonView.InitializeComponent() in C:\code\DaxStudio\src\DaxStudio.UI\Views\RibbonView.xaml:line 1

Inner Exception 1:
FileNotFoundException: Could not load file or assembly 'FontAwesome.WPF, PublicKeyToken=0758b07a11a4f466' or one of its dependencies. The system cannot find the file specified.
```

**Comment by RedTailedHawk** [comment]
Running Visual Studio 2019, and just upgraded to .NET 4.7.1.

**Comment by RedTailedHawk** [comment]
As an aside, I downloaded this so I could dig into how DaxStudio is obtaining metadata from AAS.  Specifically, I wrote a C# program to get metadata using Micosoft's `Microsoft.AnalysisServices.Tabular` assembly, and while I am seeing columns for tables, some tables only show me a small subset of columns (I am seeing some hidden and some not hidden).  But where are the missing columns?

When I run my downloaded version of DaxStudio and connect to my AAS instance, I see all the columns.  So I figured I would download the DaxStudio source and try to see how you are doing it.

But still would be nice to be able to run a debug version from Visual Studio.  Would be a good learning tool.

Thanks for any help.

**Comment by RedTailedHawk** [comment]
Ah, I guess it's the Table.Columns property vs. Table.Measures.

**Comment by dgosbell** [comment]
Maybe try using the develop branch, that's where the most up to date code is. The way we are using git at the moment is that we check in bug fixes, new features and merge pull requests into the develop branch first, then I merge develop into master when I want to do a public release.

I've actually replaced that specific package with another one that is more up to date. I had someone else have a similar issue, but it does not happen on my machines or on our AppVeyor build environment (which does a clean git clone each time). Although I had weird issues with this package in another experimental branch so there is definitely something not quite right with it, but I have not been able to isolate the problem.

> Ah, I guess it's the Table.Columns property vs. Table.Measures.

Yes in the TOM (Tabular Object Model) the Columns and Measures will be in separate collections.

In DAX Studio I am using the client library (Microsoft.AnalysisServices.AdomdClient) but I ended up build a custom CSDL parser (reading the output from the DISCOVER_CSDL_METADATA call) as this data has a bunch of extra properties that are not exposed through the standard AdomdClient classes.
